SettingsList.py

```diff
@@ -3277,6 +3277,15 @@ class SettingInfos:
         shared         = True,
     )
 
+    fast_shadow_boat = Checkbutton(
+        gui_text       = 'Fast Shadow Boat',
+        gui_tooltip    = '''\
+            The boat sequence in Shadow Temple will be massively sped up.
```

Comment:
Maybe the description should mention how this changes the stalfos fight?

Reply:
Added a line on Stalfos.